Controller.php

```diff
@@ -280,4 +281,23 @@ private function getPrettyDateForSite($datetime, $period, $timezone)
 
         return $prettyDate;
     }
+
+    private function fetchSites($alert)
+    {
+        $sites = SitesManagerApi::getInstance()->getSitesWithAtLeastViewAccess();
+
+        if (!$alert || !isset($alert['id_sites'])) {
+            return $sites;
+        }
+
+        $checked = $alert['id_sites'];
```

Comment:
Based on your previous validation this might not be an array (only checked for `isset`).